iati/core/rulesets.py

```diff
@@ -271,18 +270,41 @@ def is_valid_for(self, dataset):
         """Assert that the date value of `less` is older than the date value of `more`.
 
         Args:
-            dataset.xml_tree: an etree created from an XML dataset.
+            dataset (iati.core.Dataset): An IATI Dataset object.
 
         Return:
-            A boolean value. If `less` is older than `more`, return `True`.
+            bool: Changes depending on whether `less` is older than `more`.
```

Comment:
As this was changed from the original from containing too much implementation detail, I have simply changed it  to include implementation detail as I'm not sure how else to reach the desired specificity here.